test/framework/src/main/java/org/elasticsearch/test/ClusterServiceUtils.java

```diff
@@ -124,6 +124,15 @@ public static ClusterService createClusterService(ThreadPool threadPool, Discove
         return createClusterService(threadPool, localNode, Settings.EMPTY, clusterSettings);
     }
 
+    public static ClusterService createClusterService(ThreadPool threadPool, Settings providedSettings) {
+        return createClusterService(
+            threadPool,
+            DiscoveryNodeUtils.create("node", "node"),
+            providedSettings,
+            new ClusterSettings(Settings.EMPTY, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS)
```

Comment:
Seems a bit odd to use `Settings.EMPTY` here rather than `providedSettings` although this is what happens in other `createClusterService` overrides.